packages/dx-react-grid/docs/reference/integrated-summary.md

```diff
@@ -0,0 +1,48 @@
+# IntegratedSummary Plugin Reference
+
+A plugin that performs built-in data summary calculation.
```

Comment:
A plugin that performs built-in data summary calculation -> A plugin that performs **a** built-in data summary calculation.